knowhere/knowhere/index/vector_index/impl/bruteforce/CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
 cmake_minimum_required(VERSION 3.10)
 
-set (CMAKE_CXX_STANDARD 11)
-
-project(BruteForce)
-
-#set(CMAKE_C_COMPILER "/opt/homebrew/Cellar/llvm/13.0.0_2/bin/clang")
```

Comment:
Do not use a specific path that may not work on other laptops.

Reply:
done.